extension.neon

```diff
@@ -26,13 +26,7 @@ parameters:
         - Illuminate\Http\Request
     excludes_analyse:
         - *.blade.php
-    ignoreErrors:
-        - '#Result of function abort \(void\) is used#'
-        - '#Call to an undefined method Illuminate\\Support\\HigherOrder#'
-        - '#Method App\\Exceptions\\Handler::render\(\) should return Illuminate\\Http\\Response but returns Symfony\\Component\\HttpFoundation\\Response#'
-        - '#Property App\\Http\\Middleware\\TrustProxies::\$headers \(string\) does not accept default value of type int#'
     bootstrap: %rootDir%/../../nunomaduro/larastan/bootstrap.php
-    reportUnmatchedIgnoredErrors: false
```

Comment:
Ah, this is great!

I was using phpstan before switching to larastan a few months ago in a private project and, although I knew about this behaviour, I totally forgot about it. When I manually set this flag to true (override) I was prompted with a lot of outdated rules from my codebase! This is so hideous 😄